CONTRIBUTING.md

```diff
@@ -21,12 +21,18 @@ please ensure that:
 2. The documentation has been updated accordingly. In particular, if a
    function or class has been modified during the PR, please update the
    *docstring* accordingly.
+
+    If you are adding a new experiment or other functionality which merits
+    a tutorial, add one based on these [guidelines](docs/tutorials/GUIDELINES.md)
+   (this can be done in a separate PR)
```

Comment:
Also add the policy for who should update the tutorial after a relevant code change, once we reach a consensus.

Reply:
We agreed for the person who changes the code to also update the tutorial, unless it's an emergency code change that shouldn't be delayed, in which case they should open an issue for updating the tutorial